### DISCUSSION TITLE
Is there a console with autocompletion?

### DISCUSSION THREAD

**Discussion by halfbrained** [quodlibet/quodlibet#3727] [Q&A] [answered]
New user here. Is there a way to poke around in QL python's bowels with any kind of autocompletion? 'Python Console' plugin doesn't seem to have this, and looks like [ipython](https://github.com/quodlibet/quodlibet/issues/1847) is not in the cards

**Comment by declension** [comment, marked as answer]
No there isn't currently. Best way by far is with a good IDE, though maybe that's a different use-case to yours here.

**Reply by halfbrained** [reply]
Thanks. 

I will add it for myself to the 'Python Console' plugin, will it be welcome as a pull request? (or I shouldn't bother cleaning it up? :) )
